supplier_app/factory_boy.py

```diff
@@ -46,10 +46,10 @@ class TaxPayerArgentinaFactory(TaxPayerFactory):
     class Meta:
         model = TaxPayerArgentina
 
-    razon_social = "Monotributista"
+    business_name = "Monotributista"
     cuit = "1234569"
-    justificacion = "Justificacion falsa"
-    forma_de_pago = "Banco"
+    comments = "Justificacion falsa"
```

Comment:
`comments` variable is tied to bank account? taxPayer? should be appended to  an specific object?

Reply:
Comments is a column in the taxpayer model

Reply:
In that case it would be necessary to specify this: `taxpayer_ comments`

Reply:
I will fix that in the next PR